test/ExportVerilog/sv-dialect.mlir

```diff
@@ -233,7 +233,7 @@ rtl.module @Aliasing(%a : !rtl.inout<i42>, %b : !rtl.inout<i42>,
 }
 
 rtl.module @reg(%in4: i4, %in8: i8) -> (%a: i8, %b: i8) {
-  // CHECK-LABEL: module reg(
+  // CHECK-LABEL: module reg_0(
```

Comment:
I'd rename this test and then add a new test that is narrowly testing module keyword collision. I like to keep the tests separable in terms of functionality. What do you think?

Reply:
Good idea, updated the old test cases and added new test just for this.